charts/flink-operator/templates/proxy-role.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: {{ if .Values.namespaceScoped }}Role{{ else }}ClusterRole{{- end }}
```

Comment:
@diranged for `Role`, do we need to set namespace in `metadata`?

Reply:
No - all resources by default created by Helm charts are automatically put into a particular target namespace.. it's actually a bad practice to hard-code a `namespace` key when its not necessary.